### ISSUE TITLE
Using own template for twitter email request

### ISSUE THREAD

**Issue by iamzozo** [mmanos/laravel-social#20]
If an email is not provided and requested, it will redirect to a template. Is it possible to customize this template or use own?

**Comment by mmanos** [comment]
Yes, you should be able to publish the views in this package and then you can make changes to them. Alternatively you may customize the `user_failed_validation_redirect` config option to control where the user is redirected after the validation fails.
